baggage/baggage.go

```diff
@@ -288,12 +319,16 @@ func parseMember(member string) (Member, error) {
 		panic("failed to parse baggage member")
 	}
 
-	return Member{key: key, value: value, properties: props}, nil
+	return Member{key: key, value: value, properties: props, hasData: true}, nil
 }
 
 // validate ensures m conforms to the W3C Baggage specification, returning an
 // error otherwise.
 func (m Member) validate() error {
+	if !m.hasData {
+		return errInvalidMember
```

Comment:
Similar here, can we update this returned error to contain info what makes it invalid?

Reply:
See my other comment as well :)